tests/test_edgeql_syntax.py

```diff
@@ -759,6 +759,7 @@ def test_edgeql_syntax_name_01(self):
         SELECT foo::`bar`;
         SELECT `foo`::bar;
         SELECT `foo`::`bar`;
```

Comment:
Please add `SELECT `foo::bar` test

Reply:
I've added a syntax test with a (valid) trailing backtick in the identifier.